abr_control/arms/jaco2/config.py

```diff
@@ -105,9 +107,11 @@ def __init__(self, hand_attached=False, **kwargs):
             [-5.2974e-04, 1.2272e-02, -3.5485e-02],  # link 5 offset
             [-1.9534e-03, 5.0298e-03, -3.7176e-02]]  # joint 5 offset
         if self.hand_attached is True:  # add in hand offset
-            self.L.append([0.0, 0.0, 0.0])  # offset for the end of fingers
+            self.L.append([0.0, 0.0, 0.12])  # offset for the end of fingers
```

Comment:
I believe this is why your arm plots from `analyze` weren't aligning with the `EE`

Reply:
hmm interesting. why would we have the hand offset in master as zero? just a mistake?

Reply:
We used to handle this with the OFFSET parameter at one point, this is probably just left over from that point